src/routes/about/__tests__/get-about.e2e-spec.ts

```diff
@@ -17,8 +15,6 @@ describe('Get about e2e test', () => {
     const moduleRef = await Test.createTestingModule({
       imports: [AppModule.register()],
     })
-      .overrideModule(PostgresDatabaseModule)
```

Comment:
This mocking was previously added to prevent this initialization error but it's not longer needed.